docs/dashboard.asciidoc

```diff
@@ -3,163 +3,113 @@
 
-In *Edit Mode*, visualizations in your dashboard are stored in resizable, moveable containers.
+By default, Kibana dashboards use a light color theme. To use a dark color theme,
+click *Options* and select *Use dark theme*. To set the dark theme as the default, go
+to *Management > Advanced Settings* and set `dashboard:defaultDarkTheme`
+to `On`.
 
-[float]
-[[moving-containers]]
-==== Moving Visualizations
+[[loading-a-saved-dashboard]]
+TIP: You can import, export, and delete dashboards in *Kibana > Management >
+Saved Objects > Dashboards*.
 
-To reposition a visualization:
+[[customizing-your-dashboard]]
+== Arranging Dashboard Elements
 
-. Hover over it to display the container controls.
-. Click and hold the *Move* button in the upper right corner of the container.
-. Drag the container to its new position. 
-. Release the *Move* button.
+The visualizations and searches in a dashboard are stored in panels that you can move,
+resize, and delete.  To start editing, open the dashboard and click *Edit*
+in the menu bar.
 
-[float]
-[[resizing-containers]]
-==== Resizing Visualizations
+[role="screenshot"]
+image:images/Dashboard_Resize_Menu.png[Example dashboard]
 
-To resize a visualization:
+[[moving-containers]]
+To move a panel, click and hold the
+header of a panel and drag to the new location.
 
-. Hover over it to display the container controls.
-. Click and hold the *Resize* button in the bottom right corner of the container.
-. Drag to change the dimensions of the container. 
-. Release the *Resize* button.
+[[resizing-containers]]
+To resize a panel, click the resize control on the lower right and drag
+to the new dimensions.
 
-[float]
 [[removing-containers]]
-==== Removing Visualizations
+Additional commands for managing the
+panel and its contents are in the gear menu in the upper right.
 
-To remove a visualization from the dashboard:
+NOTE: Deleting a panel from a
+dashboard does *not* delete the saved visualization or search.
 
-. Hover over it to display the container controls.
-. Click the *Delete* button in the upper right corner of the container.
-+
-NOTE: Removing a visualization from a dashboard does _not_ delete the
-saved visualization.
-
-[float]
 [[viewing-detailed-information]]
-=== Viewing Visualization Data
+== Viewing Visualization Data
 
-To display the raw data behind a visualization:
+You can display the raw data behind a dashboard visualization using the
+Expand/Collapse button.
 
-. Hover over it to display the container controls.
-. Click the *Expand* button in the lower left corner of the container.
-This displays a table that contains the underlying data. You can also view
-the raw Elasticsearch request and response in JSON and the request statistics.
-The request statistics show the query duration, request duration, total number
-of matching records, and the index (or index pattern) that was searched.
+. Open the dashboard.
+. Hover over the visualization and click the *Expand* button in the lower
```

Comment:
This is going to be changing for 6.4 so that this functionality is exposed in the gear menu.  Fine to leave it like this for now, but I want to point out that not all visualizations support it.  Right now, in 6.3, no time series visual builder visualizations show it.  Maybe not vega/timelion... I'd have to check.  Not a huge deal, but might want to add a caveat like "If supported ...."

A lot of stuff will be going that route.  When we have "Create Machine Learning Job" it'll only be for certain embeddables.  I don't have a good generic term for embeddables.  Right now it's only saved searches and visualizations, but it could eventually be more.  In addition, even for one embeddable type (like our visualizations), panel options can change based on the visualization sub type. 

I'm not opposed to just calling them visualizations because I think that is a nice generic term in itself, but it's overloaded because right now when we say visualizations we mean a specific embeddable type. 🤷‍♀️